lib/recipient.rb

```diff
@@ -0,0 +1,65 @@
+require "httparty"
+require "dotenv"
+require_relative "workspace"
+require_relative "user"
+require_relative "channel"
+
+Dotenv.load
+
+module SlackApi
+  BASE_URL = "https://slack.com/api/chat.postMessage"
+  SLACK_TOKEN = ENV["SLACK_TOKEN"]
+
+  class SlackApiError < StandardError; end
+
+  def self.send_message(message, user)
+    user_name = User.list.map do |user_list|
+      user_list[0]
+    end
+    # array of all channel names
+    channel_name = Channel.list.map do |channel|
+      channel[0]
+    end
+
+    #### TODO: CHANGE VARIABLE NAME
+    practice = []
+    User.list.each do |user_name_list|
```

Comment:
Here you end up making the same API call multiple times, this is inefficient.  It might be much more useful to have `list` create an array of `User` instances with usernames, ids etc.